src/main/java/seedu/findvisor/model/person/PersonMeetingTodayPredicate.java

```diff
@@ -0,0 +1,57 @@
+package seedu.findvisor.model.person;
+
+import java.time.LocalDate;
+import java.util.Optional;
+
+import seedu.findvisor.commons.util.DateTimeUtil;
+import seedu.findvisor.commons.util.ToStringBuilder;
+
+/**
+ * A predicate for evaluating if a {@link Person}'s scheduled meeting at the current date.
+ * This is used to filter for persons with meetings today.
+ */
+public class PersonMeetingTodayPredicate implements PersonPredicate {
+    private final LocalDate todaysDate;
+
+    /**
+     * Constructs an {@code PersonMeetingTodayPredicate} with the today's date.
+     */
+    public PersonMeetingTodayPredicate() {
+        this.todaysDate = LocalDate.now();
+    }
+
+    /**
+     * Returns the description of this predicate, indicating the phone keyword criteria.
```

Comment:
Shoulld this be changed to:
```suggestion
     * Returns the description of this predicate, indicating the meetings occuring on the current system's date.
```